### PR TITLE
Docs: Fix formatting and broken hyperlink in UG

### PR DESCRIPTION
**Pull request by JulietTeoh**
fix errors that I introduced

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ Here are some general rules to follow when entering prefixes and parameters:
   
 * To add multiple parameters of the same prefix, add the prefix multiple times before each parameter.<br>
   e.g. To add two TAGs, enter `-t tagOne -t tagTwo`.<br>
-  e.g. To add three HEADERSs, enter `-h \"header:one\" -h \"header:two\" -h \"header:three\"`.<br>
+  e.g. To add three HEADERSs, enter `-h "header:one" -h "header:two" -h "header:three"`.<br>
```
Comment (tjtanjin) [inline docs/UserGuide.md:129]: Might be better to include whitespaces between `:` and `one`, `two` and `three` to be consistent with the header formatting.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -191,7 +191,7 @@ Check out the screenshot of each command for an idea of the expected output in t
 For URL, we do not check if your URL actually exists<br>
 If no website [protocol](#glossary-protocol) is specified, we enforce a HTTP protocol as a protocol needs to be specified for an API request to be carried out<br>
 For instance, if a user's enters `google.com` as a URL, we will prepend the URL with `http://`, making it `http://google.com`
```
Comment (tjtanjin) [inline docs/UserGuide.md:193]: Just noticed this but can help to remove the `'s` after `user`.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -211,14 +211,14 @@ Multiple headers/tags must be unique and duplicates will be ignored
 
 <div markdown="span" class="alert alert-warning">:bulb: **Tip:**
 When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative <br>
```
Comment (tjtanjin) [inline docs/UserGuide.md:213]: `person` needs to be replaced with `endpoint` here (occurring at a few other places as well).

File: docs/UserGuide.md [hunk 4]
```diff
@@ -211,14 +211,14 @@ Multiple headers/tags must be unique and duplicates will be ignored
...
-You can remove all the person’s tags by typing `t/` without specifying any tags after it
-</div><br>
+You can remove all the person’s tags by typing ` -t` without specifying any tags after it
+</div>
 
 <div markdown="span" class="alert alert-warning">:bulb: **Tip:**
 For URL, we do not check if your URL actually exists<br>
 If no website [protocol](#glossary-protocol) is specified, we enforce a HTTP protocol as a protocol needs to be specified for an API request to be carried out<br>
 For instance, if a user's enters `google.com` as a URL, we will prepend the URL with `http://`, making it `http://google.com`
```
Comment (tjtanjin) [inline docs/UserGuide.md:220]: Same `user` issue as above (and in a few other places as well).

**Review by tjtanjin — APPROVED** [review]
LGTM, just some nits to fix :)

File: docs/UserGuide.md (hunk 3 above)
Comment (JulietTeoh) [reply docs/UserGuide.md:213]: great catch

**Pull request merged**